### ISSUE TITLE
MachineImage#latestAmazonLinux is deprecated

### ISSUE THREAD

**Issue by aceat64** [Hawxy/cdk-tailscale-bastion#16]
I'm using CDK version `2.66.1`, it is generating the following warnings about the Tailscale bastions:

```
[WARNING] aws-cdk-lib.aws_ec2.MachineImage#latestAmazonLinux is deprecated.
  use MachineImage.latestAmazonLinux2 instead
  This API will be removed in the next major release.
```

I believe this is because the version of CDK (`2.20.0`) used by this package defaults to using `MachineImage.latestAmazonLinux()`. Newer versions now default to `MachineImage.latestAmazonLinux2()`, but this is not available in `2.20.0` of CDK.

**Comment by Hawxy** [comment]
I'm aware of this and was planning to deal with it sometime soon. This would be a v2 release as changing the image is a breaking change and will force the bastion to be recreated.

**Comment by Hawxy** [comment]
Fixed in #17
